src/main/resources/templates/macros/array.js.vm

```diff
@@ -0,0 +1,2 @@
+#macro(stringArray $array) [#foreach($element in $array) "$element", #end] #end
```

Comment:
This will render 

    ["something", "something", "something",] 

which is a javascript array. Unsure what I should call it.